File: B/workdir/Task0.Samples/C#/Task0.cs
using System;
using System.IO;


class Task0
{
  public const string
    InputFileName = "input.txt",
    OutputFileName = "output.txt";


  static void Main(string[] args)
  {
    StreamReader input = new StreamReader(InputFileName);
    StreamWriter output = new StreamWriter(OutputFileName);

    string[] parts = input.ReadLine().Split();
    int a = int.Parse(parts[0]),
        b = int.Parse(parts[1]);

    int c = a + b;

    output.WriteLine(c);

    input.Close();
    output.Close();
  }
}

File: B/workdir/Task0.Samples/Java/Task0.java
import java.util.*;
import java.io.*;


public class Task0
{
  public static final String
    InputFileName = "input.txt",
    OutputFileName = "output.txt";


  public static void main(String[] args) throws Exception
  {
    Scanner input = new Scanner(new File(InputFileName));
    PrintWriter output = new PrintWriter(new File(OutputFileName));

    int a = input.nextInt();
    int b = input.nextInt();

    int c = a + b;

    output.println(c);

    input.close();
    output.close();
  }
}
